src/arraymath.jl

```diff
@@ -163,3 +163,11 @@ end
         return a
     end
 end
+
+Base.min{T1<:Real,T2<:Real}(a::StaticArray{T1}, b::StaticArray{T2}) = _min(promote(a, b)...)
```

Comment:
That's reasonable. Do you want deprecation/error stubs that direct users to use `min.(a, b)` etc?